composer.json

```diff
@@ -45,12 +53,12 @@
             "vendor/bin/phpunit --verbose --configuration phpunit.xml --coverage-html tests/html-report"
         ],
         "phpcs": [
-            "vendor/bin/php-cs-fixer fix --config-file=.php_cs --verbose --diff --dry-run"
+            "vendor/bin/php-cs-fixer fix --config=.php_cs.dist --verbose --diff --dry-run"
```

Comment:
Usually config files are suffixed with `.dist` because they mean "default config". If someone wants to have their own custom file they just need to copy it and remove that suffix